packages/harmony/src/components/balance-pill/BalancePill.tsx

```diff
@@ -0,0 +1,52 @@
+import { useTheme, CSSObject } from '@emotion/react'
+
+import { Flex } from '../layout/Flex'
+import { Text } from '../text/Text'
+
+import type { BalancePillProps } from './types'
+
+/**
+ * A pill-shaped component that displays a balance, typically used to show a user's token balance.
+ */
+export const BalancePill = ({
+  balance,
+  children,
+  ...props
+}: BalancePillProps) => {
+  const { color } = useTheme()
+
+  const textStyles: CSSObject = {
+    color: color.neutral.n950
+  }
+
+  return (
+    <Flex
+      pl='s'
+      alignItems='center'
+      gap='xs'
+      borderRadius='circle'
+      border='default'
+      backgroundColor='surface1'
+      {...props}
+    >
+      <Text
+        variant='label'
+        size='s'
+        strength='strong'
+        textAlign='center'
+        css={textStyles}
```

Comment:
we may want "inverse" color here

Reply:
think we're using the v1 config so inverse isn't accessible 

![image](https://github.com/user-attachments/assets/f1b56701-15fc-4eab-8108-12e632d665a9)

Reply:
ah i see, how do you want to play this? go ahead with this for now, and follow up when v2 live?

Reply:
yea that works!